.github/workflows/deploy.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish ModuleSIte on gh-pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
I think the [checkout action](https://github.com/actions/checkout) as at v2 by now - any reason you're using v1?

Reply:
oh, I was referring to the gatsby publish template and for testing i used v1, I forgot that v2 is also available , sorry will update that

Reply:
done